tests/unit/core/test_addonManager.js

```diff
@@ -319,18 +319,6 @@ describe(__filename, () => {
         );
     });
 
-    it('should reject if addon.uninstall resolves with false', () => {
-      const fakeAddon = fakeClientAddon({
-        uninstall: sinon.stub().resolves(false),
-      });
-      fakeMozAddonManager.getAddonByID.returns(Promise.resolve(fakeAddon));
-      return addonManager
-        .uninstall('test-id', { _mozAddonManager: fakeMozAddonManager })
-        .then(unexpectedSuccess, (err) =>
-          expect(err.message).toEqual('Uninstall failed'),
-        );
-    });
```

Comment:
Not needed anymore or what?

Reply:
The [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1268075) mentioned in the comment that explains why we needed this is marked as fixed. The [commit](https://hg.mozilla.org/mozilla-central/rev/f8ee44e0d4c6) landed with that bug does appear to include a promise rejection, rather than returning a `false`, which is why this code was needed. So it looks like this code is no longer needed. I could have just fixed up the type issue, but I figured if I did that you would have added a comment asking if the bug was still open. 😛